integration-test/tests/email-autofill.macos.spec.js

```diff
@@ -151,4 +151,20 @@ test.describe('macos', () => {
         await login.selectFirstCredential(personalAddress)
         await login.assertFirstCredential(personalAddress, password)
     })
+    test.describe('matching performance', () => {
+        test('matching performance v1', async ({page}) => {
+            await forwardConsoleMessages(page)
+            await createWebkitMocks().applyTo(page)
+            await createAutofillScript()
+                .replace('isApp', true)
+                .platform('macos')
+                .applyTo(page)
+
+            await page.goto(server.urlForPath('src/Form/test-cases/usps_signup.html'))
+            const r = await performanceEntries(page, 'scanner:init')
+            for (let performanceEntry of r) {
+                console.log(performanceEntry.duration)
+            }
```

Comment:
Just so I understand, this doesn't actually check anything, it just logs the performance record so we can manually check. Is that right?

Reply:
That's correct (for now), yes

Reply:
It also ensures that the performance markers are not removed by mistake, as this test will throw in that case